### PR TITLE
Remove extra note in note shortcode

### PR DESCRIPTION
**Pull request by gavinfish**
No need to add `note` in a note shortcode.
Screen Shot 2020-02-15 at 2 41 39 PM

### CODE REVIEW

File: content/zh/docs/contribute/style/hugo-shortcodes/index.md [hunk 1]
```diff
@@ -224,7 +224,7 @@ This is **some markdown.**
 {{< tabs name="tab_with_md" >}}
 {{% tab name="Markdown" %}}
 This is **some markdown.**
-{{< note >}}**Note:** It can even contain shortcodes.{{< /note >}}
+{{< note >}}It can even contain shortcodes.{{< /note >}}
```
Comment (tengqm) [inline content/zh/docs/contribute/style/hugo-shortcodes/index.md]: can you please translate the content as well?

Comment (gavinfish) [reply content/zh/docs/contribute/style/hugo-shortcodes/index.md]: done

**Comment by tengqm** [comment]
Thanks
/lgtm

**Pull request merged**